t/04_power_action_utils.t

```diff
@@ -0,0 +1,30 @@
+use strict;
+use warnings;
+use Test::Fatal;
+use Test::More;
+use Test::Warnings;
+use Test::MockModule;
+use Test::MockObject;
+use testapi 'set_var';
+
+use power_action_utils;
+
+
+is prepare_system_shutdown, undef, 'prepare_system_shutdown has no effect by default';
+set_var('BACKEND', 'spvm');
+my @calls;
+my $mock = Test::MockModule->new('power_action_utils');
+$mock->redefine(console => sub {
+        push @calls, @_;
+        Test::MockObject->new()->set_true('kill_ssh')->set_true('disable_vnc_stalls')->set_true('stop_serial_grab');
+});
+is prepare_system_shutdown, undef, 'prepare_system_shutdown for spvm is fine';
+is_deeply \@calls, ['root-ssh'], 'root-ssh console accessed';
+@calls = [];
```

Comment:
I'd separate asserts from variable assignments and test data stuff.

Reply:
What do you mean with "separate"? Add blank lines? Normally I see them used pretty arbitrarily so I better avoid them at all :) Or do you mean something different?

Reply:
Yes, blank lines